pom.xml

```diff
@@ -611,6 +611,12 @@
                 <version>5.3.0</version>
                 <scope>test</scope>
             </dependency>
+            <dependency>
+                <groupId>org.assertj</groupId>
+                <artifactId>assertj-core</artifactId>
```

Comment:
@fhussonnois let us know if the introduction of assertj as test dependency is acceptable or whether we should refactor our unit tests.

Reply:
Hi @theworkshopcom, honnestly I would prefer to not introduce new libraries as much as possible. So, it would be nice if you could not use assertj. Thank you

Reply:
This has been addressed now